aggsender/db/migrations/0002.sql

```diff
@@ -5,7 +5,7 @@ ALTER TABLE certificate_info_history DROP COLUMN aggchain_proof;
 ALTER TABLE certificate_info_history DROP COLUMN finalized_l1_info_tree_root;
 
 -- +migrate Up
-ALTER TABLE certificate_info ADD COLUMN aggchain_proof VARCHAR;
+ALTER TABLE certificate_info ADD COLUMN aggchain_proof BLOB;
```

Comment:
@goran-ethernal should we add some default value?
i think db fetching might give an error for entries with null value

Reply:
It doesn't return an error, because I wrote a custom meddler for this field.